src/main/java/uk/gov/hmcts/darts/task/runner/impl/ProcessE2EArmRpoPendingAutomatedTask.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.darts.task.runner.impl;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.arm.service.TriggerArmRpoSearchService;
+import uk.gov.hmcts.darts.common.repository.AutomatedTaskRepository;
+import uk.gov.hmcts.darts.log.api.LogApi;
+import uk.gov.hmcts.darts.task.api.AutomatedTaskName;
+import uk.gov.hmcts.darts.task.config.AutomatedTaskConfigurationProperties;
+import uk.gov.hmcts.darts.task.runner.AutoloadingManualTask;
+import uk.gov.hmcts.darts.task.service.LockService;
+
+@ConditionalOnProperty(
+    value = "darts.automated.task.process-e2e-arm-rpo-pending.process-e2e-arm-rpo",
+    havingValue = "true"
+)
+@Slf4j
+@Component
+public class ProcessE2EArmRpoPendingAutomatedTask extends AbstractLockableAutomatedTask implements AutoloadingManualTask {
+
+    private final TriggerArmRpoSearchService triggerArmRpoSearchService;
+
+    protected ProcessE2EArmRpoPendingAutomatedTask(AutomatedTaskRepository automatedTaskRepository,
+                                                   AutomatedTaskConfigurationProperties automatedTaskConfigurationProperties,
```

Comment:
This should be the config for this task instead of the default one (So locks / users get assigned correctly)

Reply:
Good spot.